### ISSUE TITLE
Polynomial fit interactive needs buttons

### ISSUE THREAD

**Issue by jradavenport** [jradavenport/spectra#66]
There should be buttons to Increase/Decrease polynomial order, and a button to Accept

![screen shot 2016-08-02 at 11 56 07 am](https://cloud.githubusercontent.com/assets/3179868/17341538/e3953fe4-58a9-11e6-88f0-82bf458c2e83.png)

**Comment by simontorres** [comment]
What has worked for me is using events. Never tried to add buttons to matplotlib.pyplot but events seems simpler.

**Comment by jradavenport** [comment]
agreed! the workflow here is very clumsy, but my excuse is that I was optimistic I could get the pipeline working hands-free.... alas
